package.json

```diff
@@ -88,17 +88,26 @@
     "@types/react": "^18.0.18",
     "@types/react-dom": "^18.0.6",
     "@types/testing-library__jest-dom": "^5.9.1",
+    "@typescript-eslint/eslint-plugin": "latest",
     "babel-plugin-prismjs": "^2.1.0",
     "blob-polyfill": "^7.0.20220408",
-    "check-dts": "^0.6.7",
+    "check-dts": "^0.7.1",
     "csp-html-webpack-plugin": "^5.1.0",
     "customize-cra": "^1.0.0",
+    "eslint-config-airbnb-typescript-prettier": "^5.0.0",
     "eslint-import-resolver-custom-alias": "^1.3.0",
+    "eslint-plugin-import": "latest",
+    "eslint-plugin-jsx-a11y": "latest",
     "eslint-plugin-lodash-fp": "^2.2.0-a1",
+    "eslint-plugin-prettier": "latest",
+    "eslint-plugin-react": "latest",
+    "eslint-plugin-react-hooks": "latest",
     "eslint-plugin-simple-import-sort": "^7.0.0",
+    "has-bigints": "^1.0.2",
```

Comment:
What is this new dependency for?

Reply:
it's required by one of the formatters. It errors out otherwise. This is likely because we use es5 rather than ES2020, according to this post: https://futurestud.io/tutorials/eslint-how-to-fix-bigint-is-not-defined